src/components/Form/Form.tsx

```diff
@@ -102,7 +102,9 @@ export const Form = chakra(
     const alteredChildren = React.Children.map(
       children as JSX.Element,
       (child: React.ReactElement, i) => {
-        return React.cloneElement(child, { gap, id: `${id}-child${i}` });
+        return (
+          child && React.cloneElement(child, { gap, id: `${id}-child${i}` })
```

Comment:
I'm guessing this was changed for the same reason it was changed in the ButtonGroup, but not sure what that reason is.

Reply:
To make sure that child is not null/undefined before using it in `cloneElement`.